packages/react-ui-components/src/MultiSelectBox_ListPreviewSortable_DraggableListPreviewElement/multiSelectBox_ListPreviewSortable_DraggableListPreviewElement.js

```diff
@@ -89,6 +89,14 @@ export default class MultiSelectBox_ListPreviewSortable_DraggableListPreviewElem
         IconButton: PropTypes.any.isRequired
     }
 
+    handleClick = () => {
+        const {onItemClick, option} = this.props;
+
+        if (onItemClick) {
+            onItemClick(option);
+        }
```

Comment:
`onItemClick` is missing in the `propTypes` declarations:

```javascript
onItemClick: PropTypes.func,
``` 

`<MultiSelectBox_ListPreviewSortable/>` and `<MultiSelectBox/>` should also declare it.

Reply:
I agree with you about `<MultiSelectBox_ListPreviewSortable/>` but why we need it in `<MultiSelectBox/>`

Reply:
Because it passes `onItemClick` to the other components and thus accepts it as a prop like here in the ReferencesEditor:
https://github.com/neos/neos-ui/blob/3cc02a49bc64a376388792785599debfe14a8f91/packages/neos-ui-editors/src/Editors/References/index.js#L74